# 2D detection info
string class_name           # Class name
int32 class_id              # Class ID according to model
float32 confidence          # Detection confidence

# Bounding box in image coordinates
float32 x_min
float32 y_min
float32 x_max
float32 y_max